packages/server/src/synchronizer/pipeline/rules/rpc/index.ts

```diff
@@ -52,7 +53,7 @@ export const createRuleRpc: Rule = function configure({config: {src}}) {
 }
 
 export function isRpcPath(filePath: string) {
-  return new RegExp('(?:app\\/)(?!_rpc).*(?:queries|mutations)\\/.+').exec(filePath)
+  return /(?:app[\\/])(?!_rpc).*(?:queries|mutations)[\\/].+/.exec(filePath)
```

Comment:
Ha I like how the escape slash turns into the alternative char by adding a bracket 😄